src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GenericEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/order")
+public class OrderController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getOrders")
+    public List<GenericEntity> getOrders() {
+        return new ArrayList<>();
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "createOrder")
+    public void createOrder() {
+    }
+
```

Comment:
Do createOrder nie trzeba nic przekazać? żadnych argumentów ta metoda nie przyjmuje? Ja do swojej wpisałam, wyrzucić to?

Reply:
Właściwie to trzeba. Tutaj niechcący pominąłem, mogę poprawić, ale wydaje mi się, że w tym zadaniu nie ma to znaczenia, bo na razie i tak ta metoda nie ma implementacji. U siebie zostaw tak jak masz.

Reply:
Czyli GenericEntity w okrągłym nawiasie ma być i typ zwracany może być też GenericEntity. Void z tego co wiem się trudniej testuje....

Reply:
Void jako typ zwracany może zostać, ale metoda powinna mieć jakiś argument